### PR TITLE
Julia Tan Assignment 1

### PR DESCRIPTION
**Pull request by julialaurentan**

### CODE REVIEW

File: assignment1.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Julia Tan
+
+1. The 'return 0' at the end of the program is a convention that represents the success of the program. It is required for C programs.
```
Comment (oldclesleycode) [inline assignment1.txt:3]: Right - it's also needed since the main method is declared as "int main()"

File: assignment1.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+2. Instead of "%d", it should say "%f" because the variable 'x' is a float, not an integer.
```
Comment (oldclesleycode) [inline assignment1.txt:5]: Correct! You're missing a couple of more errors though - see if you can find them!

File: assignment1.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+3. "r * r" should be in parentheses so that the denominator of the fraction in the right side of the equation is "r^2" not "r". (Program reads left to right)
```
Comment (oldclesleycode) [inline assignment1.txt:7]: Correct! You're missing a couple more errors though - see if you can find them!

File: assignment1.txt [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+4. ++var is a prefix operator, while var++ is a postfix operator.
```
Comment (oldclesleycode) [inline assignment1.txt:9]: Great answer to number 4!

File: initials.c [hunk 5]
```diff
@@ -0,0 +1,15 @@
+// Julia Tan
```
Comment (oldclesleycode) [inline initials.c:1]: Perfect code! 💯 You followed all the guidelines and your code runs! Nice job! 😄

Comment (leighannehsu) [inline initials.c:1]: You're actually missing a curly brace :) but otherwise, nice job!

**Pull request closed without merging**